pages/dkp/kaptain/2.0.0/install/air-gapped-2.2/index.md

```diff
@@ -0,0 +1,96 @@
+
+    - Istio
+    - Knative (optional, if KServe is configured to work in `RawDeployment` mode)
+
+-   [`kubectl`][kubectl] on your installation machine
+
+-   For customers deploying in a multi-cluster environment (Enterprise): Ensure you have configured [Kaptain to authenticate with a Management Cluster][dex].
+
+-   Ensure the following applications are enabled in Kommander. 
+
+    Review the [Kommander installation documentation][kommander-install] for more information.
+
+<!-- TODO: Clarify which DKP doc to reference here. -->
+
+<p class="message--note"><strong>NOTE: </strong>Starting from the 1.3 release, Spark Operator is no longer installed by default with Kaptain.</p>
+
+In case you need to run Spark jobs on Kubernetes using Spark Operator, it needs to be installed separately.
+Use the following instructions to install Spark Operator from Kommander Catalog for [DKP 2.x][install-spark-dkp2].
+
+### Load the Docker images into your Docker registry
+
+1.  Download the image bundle file:
+
+    - Download `kaptain_air_gapped.tar` that will contain the required artifacts to perform an air-gapped installation.
+    - (Optional) Download the custom image artifacts `kaptain_air_gapped_cpu.tar` or `kaptain_air_gapped_gpu.tar` based on whether you need CPU or GPU for your workloads.
+
+1.  Place the bundle in a location where you can load and push the images to your private Docker registry.
+
+1.  Ensure you set the `REGISTRY_URL` and `AIRGAPPED_TAR_FILE` variable appropriately, then use the following script to load the air-gapped image bundle:
+
+   ```bash
+   dkp push image-bundle --image-bundle ${AIRGAPPED_TAR_FILE} --to-registry ${REGISTRY_URL}
+   ```
+
+   _Note: this script is slightly different than the Kommander load script and has different image registry filters._
+
+   Based on the network latency between the environment of script execution, the Docker registry, and the disk speed, this can take a while to upload all the images to your image registry.
+
+1.  Download the application and chart bundles, and extract the application bundle to the location referenced in the Kommander configuration file above.  
+
+    ```bash
+    mkdir -p kaptain-catalog-applications 
+    tar -xvf ./kaptain-catalog-applications.tar.gz --strip-components 1 -C ./kaptain-catalog-applications
+    ```
+
+1.  Load the Kaptain charts after the DKP installation with:
+
+```bash
+dkp push chart-bundle kaptain-charts-bundle.tar.gz
+```
+
+## Add Kaptain to your DKP Catalog Applications via CLI
+
+If you installed DKP with Kaptain as a workspace application in the Kommander installation file, you do not need to create a Git Repository for Kaptain.
+
+If you added Kaptain after installing DKP, you must make it available by creating a Git Repository. Use the CLI to create the GitRepository resource and add a new repository.
+
+### Create a Git repository for Kaptain
+
+<!-- TODO: use config file above -->
```

Comment:
@alembiewski @lukeogg 
I don't now how to apply this with a command, can someone provide?

Reply:
It's done with the command above.